src/app/zap-templates/zcl/data-model/chip/energy-calendar-cluster.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="Energy Management"/>
+
+  <enum name="PeakPeriodSeverityEnum" type="enum8">
+    <cluster code="0x009A"/>
+    <item name="Unused" value="0x00"/>
+    <item name="Low" value="0x01"/>
+    <item name="Medium" value="0x02"/>
+    <item name="High" value="0x03"/>
+  </enum>
+
+  <bitmap name="AuxiliaryLoadBitmap" type="bitmap8">
+    <cluster code="0x009A"/>
+    <field name="AuxiliarySwitch1" mask="0x01"/>
+    <field name="AuxiliarySwitch2" mask="0x02"/>
+    <field name="AuxiliarySwitch3" mask="0x04"/>
+    <field name="AuxiliarySwitch4" mask="0x08"/>
+    <field name="AuxiliarySwitch5" mask="0x10"/>
+    <field name="AuxiliarySwitch6" mask="0x20"/>
+    <field name="AuxiliarySwitch7" mask="0x40"/>
+    <field name="AuxiliarySwitch8" mask="0x80"/>
+  </bitmap>
+
+  <bitmap name="TransitionDayOfWeekBitmap" type="bitmap8">
+    <cluster code="0x009A"/>
+    <field name="Sunday" mask="0x01"/>
+    <field name="Monday" mask="0x02"/>
+    <field name="Tuesday" mask="0x04"/>
+    <field name="Wednesday" mask="0x08"/>
+    <field name="Thursday" mask="0x10"/>
+    <field name="Friday" mask="0x20"/>
+    <field name="Saturday" mask="0x40"/>
+  </bitmap>
+
+  <struct name="Date">
+    <cluster code="0x009A"/>
+    <item fieldId="0" name="Year" type="int16u" default="0" isNullable="true" optional="true"/>
+    <item fieldId="1" name="Month" type="int8u" min="1" max="12" default="1" isNullable="true" optional="true"/>
+    <item fieldId="2" name="Day" type="int8u" min="1" max="31" default="1" isNullable="true" optional="true"/>
+    <item fieldId="3" name="DayOfWeek" type="int8u" min="1" max="7" isNullable="true" optional="true"/>
+  </struct>
+
+  <struct name="CalendarPeriod">
```

Comment:
This name does not match the spec.

Reply:
Fixed by using alchemy tool to generate XML.